stage0/run.go

```diff
@@ -60,19 +61,19 @@ type PrepareConfig struct {
 // configuration parameters needed by Run
 type RunConfig struct {
 	CommonConfig
-	PrivateNet           bool // pod should have its own network stack
-	SpawnMetadataService bool // launch metadata service
-	LockFd               int  // lock file descriptor
-	Interactive          bool // whether the pod is interactive or not
+	PrivateNet           bool         // pod should have its own network stack
+	SpawnMetadataService bool         // launch metadata service
+	LockFd               int          // lock file descriptor
+	Interactive          bool         // whether the pod is interactive or not
+	Images               []types.Hash // application images (prepare gets them via Apps)
```

Comment:
I don't get why Run can't also use Apps, and then we could just scrap this field?